dev/cognitiveservices/data-plane/Language/generate-questionanswers.json

```diff
@@ -0,0 +1,643 @@
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "questions"
+        },
+        "responses": {
+          "200": {
+            "description": "QnA generation job result.",
+            "schema": {
+              "$ref": "#/definitions/QnAGenerationResult"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful result": {
+            "$ref": "./examples/generate-questionanswers/SuccessfulQnAGenerationJobResult.json"
+          },
+          "Successful result when documents were marked": {
+            "$ref": "./examples/generate-questionanswers/SuccessfulQnAGenerationJobResult_useAnswerMarkedInContent.json"
+          },
+          "Successful result with URL in a document": {
+            "$ref": "./examples/generate-questionanswers/SuccessfulQnAGenerationJobResult_URL.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "Supported Cognitive Services endpoint (e.g., https://RESOURCE-NAME.api.cognitiveservices.azure.com).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "format": "url",
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client API version."
+    },
+    "TopParameter": {
+      "name": "top",
+      "in": "query",
+      "description": "The maximum number of resources to return from the collection.",
+      "type": "integer",
+      "format": "int32",
+      "x-ms-parameter-location": "method"
+    },
+    "SkipParameter": {
+      "name": "skip",
+      "in": "query",
+      "description": "An offset into the collection of the first resource to be returned.",
+      "type": "integer",
+      "format": "int32",
+      "x-ms-parameter-location": "method"
+    },
+    "MaxPageSizeParameter": {
+      "name": "maxpagesize",
+      "in": "query",
+      "description": "The maximum number of resources to include in a single response.",
+      "type": "integer",
+      "format": "int32",
+      "x-ms-parameter-location": "method"
+    },
+    "JobIdParameter": {
+      "name": "jobId",
+      "in": "path",
+      "description": "Job ID.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
These are already defined in _common.json_ and you should `$ref` them from there. See other files in the Language RP folder for examples. This helps provide a unified API for Language, as well as avoids duplication issues in code generators.